data.py

```diff
@@ -0,0 +1,123 @@
+import pandas as pd
+
+class data:
+    def __init__(self,dataframe):
+        self.df = dataframe
+
+    def search(self,start_time,end_time,vehicle_type):
+        """
+        Return a data class object satisfy the three conditions:
+            1. df.start_time<start_time, 
+            2. df.finish_time<endtime, 
+            3. df.vehicle_type=vehicle_type
+       
+        Parameters
+        ----------
+        self: data 
+            data class object
+        start_time: str
+            the vehicle must enter highway after the start_time
+        end_time: str 
+            the vehicle must exit highway before the end_time
+        vehicle_type: int 
+            integer indicating the type of vehicle
+        
+        Returns
+        -------
+        Data: data
+            Data class containing the filtered data
+        """
+        filtered_df = self.df[(self.df['DerectionTime_O']>=start_time)&(self.df['DerectionTime_D']<=end_time)&(self.df['VehicleType']==vehicle_type)].copy()
+        return data(filtered_df)
+    
+    def show_n(self,n_row):
+        """
+        Sample n rows from the current dataframe
+        
+        Parameters
+        ----------
+        self: data 
+            data class object
+        n_row: int
+            number of row to sample from the dataframe
+            
+        Returns
+        -------
+        Data: data
+            Data class containing the sampled data
+        """
+        sampled_df = self.df.sample(n_row)
+        return data(sampled_df)
+
+    def sort(self,entry_number,ascending=True):
```

Comment:
Change parameter name `entry_number`  into `column_index`.
Making it more self-explaining.